### ISSUE TITLE
Benchmark error

### ISSUE THREAD

**Issue by yudataguy** [nouu-me/document_vector_search_benchmark#4]
```
    from dvsb.data.ja.jsquad_without_headline import JSQuADWithoutHeadline
ModuleNotFoundError: No module named 'dvsb.data.ja.jsquad_without_headline'
make: *** [run_benchmark] Error 1
```

Any fix?

I want to run the benchmark for the latest openai embedding models

**Comment by MAYFUNCHEN** [comment]
I have the same problem. Do you have a solution?

**Comment by MAYFUNCHEN** [comment]
> ```
>     from dvsb.data.ja.jsquad_without_headline import JSQuADWithoutHeadline
> ModuleNotFoundError: No module named 'dvsb.data.ja.jsquad_without_headline'
> make: *** [run_benchmark] Error 1
> ```
> 
> Any fix?
> 
> I want to run the benchmark for the latest openai embedding models

I have the same problem. Do you have a solution?

**Comment by TatsuyaShirakawa** [comment]
Sorry for late reply!

If you have same errors please let me know how to reproduce them.
- environment
    - OS
    - Python version
    - CPU/GPU
- command to reproduce

**Comment by TatsuyaShirakawa** [comment]
Sorry for late reply adn I found `jsquad_without_headline.py` is not commited. I'll fix soon

**Comment by TatsuyaShirakawa** [comment]
Fixed this bug #5
